tests/compass-runtime-agent/test/runtimeagent/suite.go

```diff
@@ -240,7 +239,9 @@ func (ts *TestSuite) WaitForApplicationToBeDeployed(t *testing.T, applicationNam
 			return false
 		}
 
-		return app.Status.InstallationStatus.Status == helmapirelease.Status_DEPLOYED.String()
+		logrus.Infof("Application Installation status is: %s", app.Status.InstallationStatus.Status)
+
+		return app.Status.InstallationStatus.Status == "deployed"
```

Comment:
I have already tried to do that and I concluded that it requires quite amount of work, since it forces us to port both tests to go modules, which turned out to be quite problematic - some used dependencies turned out to be not compatible. After spending some time I have decided stop loosing time for this unnecessary task and as a best workaround just hardcode this string.

Reply:
Sure ^^